config/globals.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
 	SC_THEME: process.env.SC_THEME || 'default',
 	DOCUMENT_BASE_DIR: process.env.DOCUMENT_BASE_DIR || 'https://s3.hidrive.strato.com/schul-cloud-hpi/',
+	DATABASE_AUDIT: process.env.DATABASE_AUDIT || 'true',
```

Comment:
I'd say `false` is a more sensible default value

Reply:
```suggestion
	DATABASE_AUDIT: process.env.DATABASE_AUDIT || 'false',
```